src/lib.rs

```diff
@@ -10,6 +10,7 @@ pub mod merkle;
 // RE-EXPORTS
 // ================================================================================================
 
+pub use winter_crypto::{Digest, ElementHasher, Hasher};
```

Comment:
missed on last review: do we still need to export these?

Reply:
It was a way to keep importing always from `super`, but it's not needed. Fixed!